flow/connectors/postgres/postgres.go

```diff
@@ -451,7 +451,7 @@ func syncRecordsCore[Items model.Items](
 				}
 
 				row = []any{
-					uuid.New().String(),
+					uuid.New(),
```

Comment:
should test if switching this with the old raw table syntax will break syncs
ideally we'd have a fallback or a migration

Reply:
tested with old schema, postgres can handle inserting uuid into text column